target-app/Networking/Services/TargetServices.swift

```diff
@@ -43,4 +43,23 @@ class TargetServices {
         }
     }
     
+    class func get(
+        completion: @escaping (Result<[TargetElement], Error>) -> Void
+    ) {
+        BaseAPIClient.default.request(
```

Comment:
i would make an apiClient: ApiClient and initialise this in the init as `apiClient = BaseAPIClient.default` and reuse in all the class